src/pages/dorm/reservation/discussion-room.tsx

```
@@ -0,0 +1,231 @@
+import { Button } from '@/components/ui/button'
+import { Calendar } from '@/components/ui/calendar'
+import { Input } from '@/components/ui/input'
+import { Label } from '@/components/ui/label'
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from '@/components/ui/card'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)